distribution/docs/src/main/resources/_contents/_catalog-contents/extending-catalog-contents.adoc

```diff
@@ -1255,7 +1255,8 @@ This plugin can be installed and uninstalled using the normal processes describe
 No configuration is necessary for this plugin. 
 By default, it will mark metacards with validation errors and warnings as they are reported by each metacard validator and then allow the ingest.
 To prevent the ingest of certain invalid metacards, the `Metacard Validity Marker` plugin can be configured to "enforce" one or more validators.
-Metacards that are invalid according to an "enforced" validator will not be ingested.
+The `Metacard Validity Marker` plugin can also be configured to enforce validator errors, warnings, or both.
+Metacards that are have enforced errors or enforced warnings on enforced validators will not be ingested.
```

Comment:
Yes that's how it works. Just to be sure that it's clear, enforcement of errors and warnings can be configured separately. That is, you can set errors to be enforced but not warnings, warnings but not errors, both, or neither